### ISSUE TITLE
Does your method need ground removal?

### ISSUE THREAD

**Issue by zwqnju** [emecercelik/ssl-3d-detection#2]
As I know, SceneFlow method usually needs to remove ground points first.

**Comment by MingyuLiu1** [comment]
Nope, we did not remove the ground points, instead we randomly sample n points from the raw point cloud as input to the scene flow network.
